tests/e2e/tests/critical-path/database/clone-databases.e2e.ts

```diff
@@ -37,6 +38,7 @@ test
         await clickOnEditDatabaseByName(ossStandaloneConfig.databaseName);
         // Verify that user can cancel the Clone by clicking the “Cancel” or the “x” button
         await t.click(addRedisDatabasePage.cloneDatabaseButton);
+        await browserPage.verifyDatabaseStatusIsVisible();
```

Comment:
you need to add test cases names from checklist (which are marked as E2E) to each test
e.g. here you forgot to add comment:
// Verify new connection badge for cloned database

Reply:
and please add all other tests from checklist as comments to be able to find tests in code